packages/components/src/truncate/types.ts

```diff
@@ -17,19 +29,22 @@ export interface Props {
 	 *
 	 * @default 'auto'
 	 */
-	ellipsizeMode?: TruncateEllisizeMode;
+	ellipsizeMode?: TruncateEllipsizeMode;
 	/**
 	 * Determines the max characters when `truncate` is set.
 	 *
 	 * @default 0
 	 */
 	limit?: number;
```

Comment:
In a similar way to the previous comment regarding `numberOfLines`, I'm not sure that `0` is a sensible default for `limit` either:

- `Infinity` could be a better value, if we wanted to have a default value that doesn't actually cause truncation?
- with the current implementation, limit needs to be a value greater than `2` in order to have any effect

I understand that this PR should be only a refactor to TypeScript, so feel free to ignore these comments for the sake of this PR — I'm mostly writing notes down in case they can be useful in the future.

Reply:
👍 I'm not sure what the best approach is here. Feel free to push changes if you have any.

With all the other changes maybe we should do it in a separete pull request?